sql/src/main/java/io/crate/operation/collect/sources/InformationSchemaIterables.java

```diff
@@ -65,32 +66,32 @@ public InformationSchemaIterables(final Schemas schemas,
         sqlFeatures = new SqlFeaturesIterable();
     }
 
-    public Iterable<SchemaInfo> schemas() {
-        return schemas;
+    public CompletableFuture<Iterable<SchemaInfo>> schemas() {
```

Comment:
I'd tend to wrap them on the call-site instead - allows for more flexibility in how they're used. E.g. in a test it isn't necessary to unwrap the iterable.